### PR TITLE
Removing hardcoded urls for custom build from CI

### PR DESCRIPTION
**Pull request by zmhassan**
Using customer build args we can do custom builds 

```bash
docker build -t openshift-spark --build-arg DISTRO_LOC=https://dist.apache.org/repos/dist/release/spark/spark-2.1.0/spark-2.1.0-bin-hadoop2.7.tgz --build-arg DISTRO_NAME=spark-2.1.0-bin-hadoop2.7
```

### CODE REVIEW

**Comment by mattf** [comment]
make sure this continues to build in travis and the autogenerated docker hub build (https://hub.docker.com/r/radanalyticsio/openshift-spark/)

**Comment by zmhassan** [comment]
I added a change to .travis.yml to fix that

**Comment by mattf** [comment]
what about the docker hub autobuilds? maybe you can default the --build-args?

**Comment by zmhassan** [comment]
Good point! I'll add a default which is going to be the original url we put in there originally.

**Comment by mattf** [comment]
ok, be sure to revert the .travis.yml so travis builds essentially the same way docker hub will

**Comment by zmhassan** [comment]
Good point. I've reverted and committed.

File: .travis.yml [hunk 1]
```diff
@@ -6,4 +6,4 @@ services:
   - docker
 
 script:
-  - make build
+  - make build
```
Comment (mattf) [inline .travis.yml]: should be no change here

**Comment by zmhassan** [comment]
@mattf I've reverted travis.yml .

**Comment by mattf** [comment]
@tmckayus ptal

**Pull request merged**